applications/fastsum/fastsum.c

```diff
@@ -582,22 +590,22 @@ static C SearchBox(R *y, fastsum_plan *ths)
   }
   else
   {
-    exit(EXIT_FAILURE);
+    return 0.0/0.0; //exit(EXIT_FAILURE);
```

Comment:
There does not seem to be a portable way to define a NaN. Even C99 doesn't require systems to support them. Can't we just return zero and document that the result is undefined for dimensions larger than 3?